lib/src/http/http_body.dart

```diff
@@ -56,8 +56,14 @@ class HttpBody extends BaseHttpBody {
   Encoding _encoding;
 
   /// Construct the body to an HTTP request or an HTTP response from bytes.
+  ///
+  /// If an encoding cannot be parsed from the content-type header (via the
+  /// `charset` param), then [fallbackEncoding] will be used (UTF8 by default).
   HttpBody.fromBytes(MediaType this.contentType, List<int> bytes,
-      {Encoding fallbackEncoding}) {
+      {Encoding fallbackEncoding: UTF8}) {
+    if (fallbackEncoding == null) {
+      throw new ArgumentError.notNull('fallbackEncoding');
```

Comment:
Would it be okay to set it to `UTF8` if it is null here instead of throwing an error?